galaxy/main/models/importing.py

```diff
@@ -18,18 +18,11 @@ class Meta:
     STATE_FAILED = constants.ImportTaskState.FAILED.value
     STATE_SUCCESS = constants.ImportTaskState.SUCCESS.value
 
-    collection = models.ForeignKey(
```

Comment:
Importing repositories breaks and I suspect it's because this field was removed. I think the collection field needs to be removed from the importtask serializer.

```
2019-04-02 14:56:18,651 ERROR django.request: Internal Server Error: /api/v1/imports/
Traceback (most recent call last):
  File "/usr/share/galaxy/venv/lib64/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/usr/share/galaxy/venv/lib64/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/share/galaxy/venv/lib64/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/share/galaxy/venv/lib64/python3.6/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/usr/share/galaxy/venv/lib64/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/share/galaxy/venv/lib64/python3.6/site-packages/rest_framework/views.py", line 494, in dispatch
    response = self.handle_exception(exc)
  File "/usr/share/galaxy/venv/lib64/python3.6/site-packages/rest_framework/views.py", line 454, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/usr/share/galaxy/venv/lib64/python3.6/site-packages/rest_framework/views.py", line 491, in dispatch
    response = handler(request, *args, **kwargs)
  File "/galaxy/galaxy/api/views/views.py", line 440, in post
    response = {'results': [serializer.data]}
  File "/usr/share/galaxy/venv/lib64/python3.6/site-packages/rest_framework/serializers.py", line 537, in data
    ret = super(Serializer, self).data
  File "/usr/share/galaxy/venv/lib64/python3.6/site-packages/rest_framework/serializers.py", line 262, in data
    self._data = self.to_representation(self.instance)
  File "/usr/share/galaxy/venv/lib64/python3.6/site-packages/rest_framework/serializers.py", line 487, in to_representation
    fields = self._readable_fields
  File "/usr/share/galaxy/venv/lib64/python3.6/site-packages/django/utils/functional.py", line 37, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/usr/share/galaxy/venv/lib64/python3.6/site-packages/rest_framework/serializers.py", line 377, in _readable_fields
    field for field in self.fields.values()
  File "/usr/share/galaxy/venv/lib64/python3.6/site-packages/rest_framework/serializers.py", line 363, in fields
    for key, value in self.get_fields().items():
  File "/galaxy/galaxy/api/serializers/serializers.py", line 94, in get_fields
    ret = super(BaseSerializer, self).get_fields()
  File "/usr/share/galaxy/venv/lib64/python3.6/site-packages/rest_framework/serializers.py", line 1022, in get_fields
    source, info, model, depth
  File "/usr/share/galaxy/venv/lib64/python3.6/site-packages/rest_framework/serializers.py", line 1167, in build_field
    return self.build_unknown_field(field_name, model_class)
  File "/usr/share/galaxy/venv/lib64/python3.6/site-packages/rest_framework/serializers.py", line 1279, in build_unknown_field
    (field_name, model_class.__name__)
django.core.exceptions.ImproperlyConfigured: Field name `collection` is not valid for model `ImportTask`.
2019-04-02 14:56:18,656 ERROR django.server: "POST /api/v1/imports/ HTTP/1.1" 500 24867
```